app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
$ bundle exec rails -v
Expected string default value for '--rc'; got false (boolean)
Rails 4.1.7

I suspect we'll want to get you an account on the server so you can clean up this mess :).